index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+la<!DOCTYPE html>
```

Comment:
This is the one thing I don't approve of in this text. 😉

Reply:
Lol, I would have caught it eventually...